$NetBSD: patch-CMakeLists.txt,v 1.1 2016/03/29 10:55:45 markd Exp $

Don't add doc subdirectory twice - fixes build with
latest cmake
--- CMakeLists.txt.orig	2014-11-04 18:43:47.000000000 +0000
+++ CMakeLists.txt
@@ -351,5 +351,5 @@ macro_display_feature_log()
 include(MacroOptionalAddSubdirectory)
 macro_optional_add_subdirectory( po )
 
-include(MacroOptionalAddSubdirectory)
-macro_optional_add_subdirectory( doc )
+#include(MacroOptionalAddSubdirectory)
+#macro_optional_add_subdirectory( doc )
